internal/security/config/command/proxy/adduser/command_test.go

```diff
@@ -52,24 +52,47 @@ func TestAddUserBadArg(t *testing.T) {
 
```

Comment:
in the `NewCommandTest` should we add the testing for the required `--jwt` case?

Reply:
add --jwt required test case

Reply:
Ok - working on that now... 👍